From: Stephan Bergmann <sbergman@redhat.com>
Date: Mon, 4 Dec 2017 21:33:35 +0000 (+0100)
Subject: Condition for definition of LastIdent must match condition for its use
X-Git-Url: https://gerrit.libreoffice.org/gitweb?p=core.git;a=commitdiff_plain;h=68f8e48e8c264056d60628da488404348f9f2f17

Condition for definition of LastIdent must match condition for its use

Change-Id: Iccc58d396d8c80a717e9d33ed8d872aadea08afa
---

diff --git a/sal/osl/unx/thread.cxx b/sal/osl/unx/thread.cxx
index 639e8b6..affbc5c 100644
--- a/sal/osl/unx/thread.cxx
+++ b/sal/osl/unx/thread.cxx
@@ -567,7 +567,7 @@ static int HashSize = SAL_N_ELEMENTS(HashTable);
 
 static pthread_mutex_t HashLock = PTHREAD_MUTEX_INITIALIZER;
 
-#if ! (defined LINUX || defined MACOSX || defined IOS)
+#if ! ((defined LINUX && !defined __FreeBSD_kernel__) || defined MACOSX || defined IOS)
 static oslThreadIdentifier LastIdent = 0;
 #endif
 
